scripts/upgrade

```diff
@@ -62,6 +62,34 @@ exec_occ() {
     php$NEXTCLOUD_PHP_VERSION --define apc.enable_cli=1 occ --no-interaction --no-ansi "$@")
 }
 
+#=================================================
+# HANDLE DATABASE MIGRATION FROM MYSL TO PSQL
+#=================================================
+
+mysql_db_pwd=$(ynh_app_setting_get --app=$app --key=mysqlpwd)
+
+if [ -n "$mysql_db_pwd" ]
```

Comment:
I'm not sure we can really rely on the mysqlpwd ... It's supposed to be automatically deleted and renamed to `db_pwd` during the v1->v2 transition (so before running this script), cf https://github.com/YunoHost/yunohost/blob/dev/src/utils/resources.py#L1366-L1372

Reply:
what about sonthing like Mattermost  https://github.com/YunoHost-Apps/mattermost_ynh/blob/52d95b9785edbe71e84fb9b99a26bec68544f123/scripts/upgrade#L40